pyqube/events/handlers.py

```diff
@@ -234,3 +235,13 @@ def on_ticket_called(
             topic += f"/counters/{counter_id}/tickets/called"
 
         return self.add_mqtt_handler(topic, AnsweringTicket)
+
+    def on_ticket_changed_state(self):
+        """
+        Registers a handler for the 'changed-state' event of tickets.
+
+        Returns:
+            The decorator for the handler function.
+        """
+        topic = f"locations/{self.location_id}/tickets/+/changed-state"
```

Comment:
Did the `+` worked because of the api rules?

Reply:
Yes. I ran a code with decorator like example added and it worked.